Framework/PythonInterface/mantid/api/src/Exports/AnalysisDataService.cpp

```diff
@@ -1,7 +1,14 @@
+#include "MantidPythonInterface/kernel/Converters/CloneToNumpy.h"
```

Comment:
Where is the `CloneToNumpy` used?

Reply:
Good catch. I attempted this a different way at first and then backed out.